deploy/cr.yaml

```diff
@@ -171,6 +171,10 @@ spec:
         cpu: "300m"
         memory: "0.5G"
     volumeSpec:
+#      annotations:
```

Comment:
@pooknull I think we need to move it under `persistentVolumeClaim:`  So, it should be like:
```
volumeSpec:
  persistentVolumeClaim:
    annotations: {}
    labels: {}
```
I have discussed it with @spron-in and he agrees with it.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1096/commits/9401698f2f4b4176a21dccff943eb01c2e7b632a